integration-tests/src/tests/client/features/in_memory_tries.rs

```diff
@@ -251,6 +280,20 @@ fn get_block_producer(env: &TestEnv, head: &Tip, height_offset: u64) -> AccountI
     block_producer
 }
 
+fn check_block_does_not_have_missing_chunks(env: &TestEnv, block: &Block) {
+    if block.header().prev_hash() != env.clients[0].chain.genesis_block().hash() {
```

Comment:
Can't we use `chunk.height_included` and `block.height` here? Why do we need to fetch the prev_block?

Reply:
Also nit: can pass just client into function instead of TestEnv and can incorporate early return for block after genesis.

Reply:
+1, we recently introduced `ShardChunkHeader::is_new_chunk` for that

Reply:
Haha I was still not confident in my ability to correctly use the heights to compare, that was why :P